FindMe/Location/LocationManager.m

```diff
@@ -109,11 +112,14 @@ - (void)beginTracking
     }
 }
 
-- (void)stopTracking
+- (void)stopTracking:(NSDate *)startDate
```

Comment:
I don't get this. Tracking, which should be named "Trace" or "Trail", has:

```
@property (nonatomic, strong) NSDate *dateStarted;
@property (assign) int numberOfContacts;
@property (nonatomic, strong) NSArray *locations;
@property (nonatomic, strong) NSString *duration;
```

So why doesn't this function call updateTrace and passing in a trace?